src/test/resources/META-INF/jahia-content-editor-forms/overrides/forms/nt_base.hidden-layout.json

```diff
@@ -0,0 +1,11 @@
+{
+  "nodeType": "nt:base",
+  "hasPreview": true,
+  "priority": 3.0,
+  "sections": [
+    {
+      "name": "layout",
+      "hide:": "true"
+    }
+  ]
+}
```

Comment:
Does this overrides will apply ?
If it's just used for testing purpose it should be in test folder and not in the project sources.